### PR TITLE
fix(THEEDGE-3757): add validation to imageset

### PR DESCRIPTION
**Pull request by acosferreira**
Fix validation to avoid exception for navigate.
Check if deviceImageSet is filled before find the imae value
THEEDGE-3757

### CODE REVIEW

**Comment by adonispuente** [comment]
I couldnt find the ticket referenced in the commit message, and as someone who isnt familiar with the groups page im not sure what im looking for in the video. Could you provide more insight into what this change is for? 
Or maybe @mkholjuraev or @gkarat have a better idea

File: src/components/GroupSystems/GroupImmutableSystems.js [hunk 1]
```diff
@@ -187,7 +187,7 @@ const GroupImmutableSystems = ({ groupName, groupId, ...props }) => {
 
   //enable disable bulk update based on selection, must refactor
   useEffect(() => {
-    if (selected.size > 0) {
+    if (selected.size > 0 && deviceImageSet?.size > 0) {
```
Comment (LiorKGOW) [inline src/components/GroupSystems/GroupImmutableSystems.js:190]: Shouldn't `deviceImageSet` be added as a dependency to the useEffect's dependency array ? 🤔 
The condition could change when it is changed, which would effect the Effect

**Review by LiorKGOW — CHANGES_REQUESTED** [review]

File: src/components/GroupSystems/GroupImmutableSystems.js (hunk 1 above)
Comment (acosferreira) [reply src/components/GroupSystems/GroupImmutableSystems.js:191]: @LiorKGOW included the validation on useEffect but I kept the condition cause we should not get in this validation if there is no imageSet

**Review by LiorKGOW — DISMISSED** [review]
Unfortunately I can't reproduce the bug that was mentioned on the ticket
Could someone have a look ?
@RedHatInsights/team-interact

**Review by LiorKGOW — COMMENTED** [review]
Unfortunately I can't reproduce the bug that was mentioned on the ticket
Could someone have a look ?
@RedHatInsights/team-interact

**Review by LiorKGOW — APPROVED** [review]
Works fine to me 
Thank you @acosferreira

**Pull request merged**